contracts/Core/RewardManager.sol

```diff
@@ -81,6 +88,12 @@ contract RewardManager is Initializable, Constants, RewardManagerParams, IReward
         }
     }
 
+    /**
+     * @dev Penalties are given to stakers based their activity if they have been inactive for more than the grace period
+     * and their votes in the previous epoch compared to the medians confirmed. Penalties on botes depend upon how far were
```

Comment:
I don't want to add review on English 😄 
But here it should be ``votes`` instead of ``botes``

Reply:
fixed https://github.com/razor-network/contracts/pull/712/commits/3300506664f50da270755398db4cd653542c4b21